lua/plugins.lua

```diff
@@ -68,11 +68,12 @@ return {
     -- event = "InsertEnter",
     after = "nvim-compe",
     config = function()
-      require "core.autopairs"
+      require("core.autopairs").setup()
       if lvim.builtin.autopairs.on_config_done then
         lvim.builtin.autopairs.on_config_done(require "nvim-autopairs")
       end
     end,
+    disable = not lvim.builtin.autopairs.active,
```

Comment:
While at it, would you extend this to other builtins ?

Reply:
Tracked in #1318